casper/src/main/scala/coop/rchain/casper/engine/Running.scala

```diff
@@ -57,13 +57,15 @@ object Running {
       _ <- engine.withCasper(
             casper => {
               for {
-                tipHash      <- MultiParentCasper.forkChoiceTip[F](casper)
```

Comment:
Why not update this function instead of duplicating code in both places where this function was used?

Reply:
Because it's not fork choice tip - we need all tips. But looks like this function is not used anywhere else... Well, this is to my question why at all we need  to know fork choice tip. So might be we can remove/ rename the message. But I wanted this PR to not touch much

Reply:
I removed this function at all in <https://github.com/rchain/rchain/pull/3414/commits/0d3c799ac0be4643549554491d0c986e99174b1f>